src/checks.jl

```diff
@@ -340,7 +340,7 @@ end
 """
     check_all_qualified_accesses_are_public(mod::Module, file=pathof(mod); ignore::Tuple=(),
                                             skip::$(TUPLE_MODULE_PAIRS)=(Base => Core,),
-                                            allow_internal_accesses=true)
+                                            from::Tuple=(), allow_internal_accesses=true)
```

Comment:
Actually, I don't like using the empty set as a sentinel here, because it has the wrong semantics; empty set should not check anything. That seems pedantic at first but `from` could be programmatically generated and happen to come up empty in some situation where you don't want to check anything. SO let's make the default `nothing`, then change the check from `isempty` to `isnothing`. Then whenever `from` is not `nothing`, we will check it. We will also allow vectors and stuff, not just tuples.
```suggestion
                                            from=nothing, allow_internal_accesses=true)
```